src/DistributedObjectEvents.md

```diff
@@ -115,6 +115,9 @@ Entry Removed:EntryEvent{entryEventType=REMOVED, member=Member [192.168.1.100]]:
 ![image](images/NoteSmall.jpg) ***NOTE:*** *Please note that the method `IMap.clear()` does not fire an "EntryRemoved" event, but fires a "MapCleared" event.*
 <br></br>
 
+<br></br>
+![image](images/NoteSmall.jpg) ***NOTE:*** *A listener runs on the event threads that may also be used by the other listeners. So, you should NEVER block these event threads.*
```

Comment:
I think the language can still be more imperative. 

Something along these lines: "A listener is not allowed to block a thread. Blocking operations have to be offloaded to another thread(pool)"